openshift-template.yaml

```diff
@@ -190,6 +195,15 @@ resources:
           port_range_min: 22
           port_range_max: 22
 
+  http_secgroup:
+    type: OS::Neutron::SecurityGroup
+    properties:
+      rules:
+        - protocol: tcp
+          remote_ip_prefix: 0.0.0.0/0
+          port_range_min: 80
+          port_range_max: 80
```

Comment:
Shouldn't this be configured to run over https?

Reply:
Yes it will be once the work is done to move zabbix to https in the ansible deployment. For now its http.